week_2/pet.rb

```diff
@@ -46,13 +46,19 @@ def initialize(name: , animal_type_id: , food_consumed_per_day:)
 
   # Return the habitat of the pet
   def habitat
-    raise NotImplementedError # TODO
+    if @animal_type_id in 1..3
+      return "Aquatic"
+    elsif @animal_type_id == 4
+      return "Terrariums"
+    elsif @animal_type_id in 5..6
+      return "Cages"
+    end
```

Comment:
If a new HABITAT is to be added, here, both the `HABITATS` hash and this function has to be updated. Which is not convenient. So, make use of the `HABITATS` hash to find the habitat here.

Reply:
So.. can you please tell me how to do that?..